lib/sml-conv/conversionals.sig

```diff
@@ -1,14 +1,47 @@
+(* This the payoff for creating a CONV ascribing structure. This signature
+ * specifies several "generic" conversions as well as ways to combine
+ * existing ones.
+ *)
 signature CONVERSIONALS =
 sig
   type conv
 
+  (* CID is the conversion which always succeeds but does nothing.
+   * It simply returns the supplied term with no modifications
+   *)
   val CID : conv
+
```

Comment:
The documentation of the conversionals should not refer to the application of a conversional to some term, since the `conv` type is abstract. I'm not sure what the consequences of this are for documenting, but probably we want the explanations of the conversionals to only refer to terms which are understood in the context of the signature.